setup.py

```diff
@@ -58,7 +58,8 @@
         'ipywidgets>=7.6.0,<9',
         'traittypes>=0.2.1,<3',
         'xyzservices>=2021.8.1',
-        'branca>=0.5.0'
+        'branca>=0.5.0',
+        'pandas>=1.4.4'
```

Comment:
We don't need pandas as a dependency

```suggestion
        'branca>=0.5.0'
```

Reply:
If that was for fixing the ui tests (that needs pandas), you should instead install pandas in test environment.